### ISSUE TITLE
IniFileSettings can only save into %appdata%

### ISSUE THREAD

**Issue by adongy** [lxn/walk#249]
Hi,

I was looking at the IniFileSettings implementation of Settings https://github.com/lxn/walk/blob/master/inifilesettings.go and noticed that the path of the saved file is "hardcoded": it will always be saved into the %appdata% directory of the user.

This is useful for moving the application without losing settings, but I'm trying to make a portable binary. As such, I'd like a parameter (a boolean ?) that would allow me to specify that the ini file i'm trying to create is relative to the current directory.

In short, it would be a new boolean in the IniFileSettings portable (which would then default to false in the factory), as well as a new function
```
func (ifs *IniFileSettings) SetPortable(portable bool) {
    ifs.portable = portable
}
```

That way, it's backwards compatible.

Let me know if you'd like a PR.

**Comment by lxn** [comment]
Sounds reasonable, but you should also add a `Portable` getter.

Please go ahead, and don't forget to add yourself to the AUTHORS file :-)
